reviews/views.py

```diff
@@ -77,6 +77,83 @@ def get_context_data(self, **kwargs):
         context['data_url'] = reverse("reviews:api:open", args=[self.committee])
 
         return context
+    
+
+class CommitteeMembersWorkloadView(GroupRequiredMixin, CommitteeMixin, generic.TemplateView):
+    #TODO: Look into improving the should_end_date variable. See notes
+
+    template_name = 'reviews/committee_members_workload.html'
+    group_required = [settings.GROUP_SECRETARY]
+    
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context['decisions'] = self.get_all_open_decisions()
+        context['today'] = date.today()
+        context['counts_dict'] = self.get_review_counts_last_year
+
+        return context
+
+    def get_all_open_decisions(self):
+        '''Returns a queryset with all open decisions'''
+
+        objects = Decision.objects.filter(
+            # This fetches all Decisions which are not approved or need revision
+            go__exact = '',
+            review__proposal__reviewing_committee = self.committee,
+            ).select_related(
+            'reviewer',
+            'review',
+            'review__proposal',
+            ).order_by(
+            "reviewer",
+            "review__date_start"
+            )
+
+        return objects
+    
+    def get_review_counts_last_year(self):
+        '''This method creates a dictionary with an overview of all reviews of
+        the past year for each reviewer, by trajectory type (short route,
+        long route, or revision).'''
+
+        dict={}
+        one_year_ago = date.today() - timedelta(days=365)
```

Comment:
Django will complain that you are using a timezone 'naive' date here. (Well, dates are always naive, but Django will transform your date to  a datetime as `date_start` is a datetime)

In reality it does not matter, as the timezone is not going to cause meaningful problems here; But to keep Django from complaining it's probably best to make this a timezone aware datetime:
```python
from django.utils import timezone
[..]
one_year_ago = timezone.now() - timedelta(days=365)
```